### PR TITLE
Render Handbook maintainers

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Allow users to understand who should review changes to that page and integrates with the review process.

### CODE REVIEW

File: src/handbook/handbook.11tydata.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const codeowners = require('codeowners');
+
```
Comment (ZJvandeWeg) [inline src/handbook/handbook.11tydata.js:2]: 💥 

```suggestion
```

File: src/handbook/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 navTitle: Handbook
+maintainer: zeger-jan-van-de-weg
```
Comment (ZJvandeWeg) [inline src/handbook/index.md:3]: 💥 
```suggestion
```

File: .eleventy.js [hunk 3]
```diff
@@ -221,7 +241,22 @@ module.exports = function(eleventyConfig) {
             callback(null, code);
         }
     });
-    
+
+    eleventyConfig.addShortcode("renderTeamMember", function(teamMember) {
+        // When the author is no longer at FlowForge
+        if (typeof teamMember === "undefined") {
+				    return ""
```
Comment (ZJvandeWeg) [inline .eleventy.js:248]:
```suggestion
            return ""
```

File: .eleventy.js [hunk 4]
```diff
@@ -221,7 +241,22 @@ module.exports = function(eleventyConfig) {
...
+			  }
```
Comment (ZJvandeWeg) [inline .eleventy.js:249]:
```suggestion
    }
```

File: src/handbook/handbook.11tydata.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    eleventyComputed: {
+        maintainer: data => new codeowners().getOwner(data.filePath)
```
Comment (ZJvandeWeg) [inline src/handbook/handbook.11tydata.js:3]:
```suggestion
        maintainers: data => new codeowners().getOwner(data.filePath)
```

**Review by joepavitt — APPROVED** [review]

**Pull request merged**